library/items/item_group/src/main/java/org/quiltmc/qsl/itemgroup/mixin/client/ItemGroupMixin.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.itemgroup.mixin.client;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.item.ItemGroup;
+
+import org.quiltmc.qsl.itemgroup.impl.QuiltCreativePlayerInventoryScreenWidgets;
+
+@Mixin(ItemGroup.class)
+public abstract class ItemGroupMixin {
+	@Shadow
+	public abstract int getIndex();
+
+	@Shadow
+	public abstract boolean isTopRow();
+
+	@Inject(method = "isTopRow", cancellable = true, at = @At("HEAD"))
+	private void isTopRow(CallbackInfoReturnable<Boolean> info) {
+		if (getIndex() > 11) {
```

Comment:
```suggestion
		if (this.getIndex() > 11) {
```

For readability, took me a little while to figure out where this comes from.